packages/core/src/middleware/koa-log.ts

```diff
@@ -90,7 +91,11 @@ export default function koaLog<
     try {
       await next();
     } catch (error: unknown) {
-      logger.set({ result: LogResult.Error, error: String(error) });
+      const errorInfo = error instanceof RequestError ? JSON.stringify(error.body) : String(error);
+      logger.set({
+        result: LogResult.Error,
+        error: errorInfo,
```

Comment:
do we log error code and message?

Reply:
Have added `error code` and `message` in the error logs in this PR.

Maybe need further discussion?

Reply:
Refactored.

See Slack thread [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1654590201727249?thread_ts=1654570553.888139&cid=C02A8G4HVAM)